content/en/docs/setup/install/istioctl/index.md

```diff
@@ -83,6 +86,12 @@ to install the `demo` profile:
 $ istioctl install --set profile=demo
 {{< /text >}}
 
+## Check what's installed
+
+`Istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can
```

Comment:
I believe `istioctl` should be lower case the same as with your Note above.

Reply:
Thanks, done.